src/arrayMethodJoin.js

```diff
@@ -4,25 +4,18 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     let result = '';
```

Comment:
What is it, result? please rewrite a name of your variable

Reply:
Agree with @Jamal-Amoian, result is bad name 💩